pyfstat/tcw_fstat_map_funcs.py

```diff
@@ -612,6 +716,11 @@ def pycuda_compute_transient_fstat_map(multiFstatAtoms, windowRange):
         )
     )
 
+    if BtSG:
```

Comment:
```suggestion
    if BtSG: # pragma: no cover
```

Reply:
Rejecting this one, this whole function _should_ be covered, including this case.